app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/labtest/LabTestDetailFragment.kt

```diff
@@ -44,67 +45,99 @@ class LabTestDetailFragment : BaseRecordDetailFragment(R.layout.fragment_lab_tes
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        setupComposeToolbar(binding.composeToolbar.root)
-        setUpRecyclerView(args.hdid)
-        viewModel.getLabTestDetails(args.labOrderId)
-        observeUiState()
         observePdfData()
-        initComments()
+        observeCommentsSyncCompletion()
     }
 
-    override fun getCommentView(): AddCommentLayout = binding.comment
+    override fun onResume() {
+        super.onResume()
+        viewModel.getLabTestDetails(args.labOrderId)
+    }
 
-    override fun getScrollableView() = binding.rvLabTestDetailList
+    @Composable
+    override fun GetComposableLayout() {
 
-    override fun getCommentEntryTypeCode() = CommentEntryTypeCode.LAB_RESULTS
+        val uiState = viewModel.uiState.collectAsState().value
```

Comment:
Please use collectAsStateWithLifecycle instead of collectAsState.
Reference: https://medium.com/androiddevelopers/consuming-flows-safely-in-jetpack-compose-cde014d0d5a3

Reply:
done

Reply:
done